scripts/get_medium.py

```diff
@@ -75,7 +75,22 @@ def main():
     Retrieves latests RAPIDS posts from Medium's RSS feed and writes the content
     to a YAML file that can be used by Hugo.
     """
-    response = requests.get("https://medium.com/feed/rapids-ai")
+    headers = {
+        "accept": "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.7",
+        "accept-language": "en-US,en;q=0.9",
+        "cache-control": "max-age=0",
+        "priority": "u=0, i",
+        "sec-ch-ua": '"Chromium";v="124", "Google Chrome";v="124", "Not-A.Brand";v="99"',
+        "sec-ch-ua-mobile": "?0",
+        "sec-ch-ua-platform": '"macOS"',
+        "sec-fetch-dest": "document",
+        "sec-fetch-mode": "navigate",
+        "sec-fetch-site": "none",
+        "sec-fetch-user": "?1",
+        "upgrade-insecure-requests": "1",
+        "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36",
+    }
```

Comment:
I tried this locally in a devcontainer and I didn't need anything except a `user-agent`. This worked for me:
```suggestion
    headers = {
        "user-agent": "Mozilla/5.0",
    }
```